contracts/OpenSTValue.sol

```diff
@@ -380,20 +380,20 @@ contract OpenSTValue is OpsManaged, Hasher {
 		
 		Unstake storage unstake = unstakes[_redemptionIntentHash];
 
-    	// require that the unstake has expired and that the redeemer has not
-    	// processed the unstaking, ie unstake has not been deleted
-    	require(unstake.expirationHeight > 0);
-    	require(unstake.expirationHeight <= block.number);
+			// require that the unstake has expired and that the redeemer has not
```

Comment:
we will need to discuss a standard and then implement a Linter check in the CI (to be put in place); for now OK, but the editors fighting is really not clean on the origin repo

Reply:
Hey Ben, me, deepesh and abhay have synched are tab space to two. We can decide on which linter to use as we all use different editors for coding.